extensions/rich_text_components/Image/Image.py

```diff
@@ -26,6 +26,7 @@ class Image(base.BaseRichTextComponent):
     frontend_name = 'image'
     tooltip = 'Insert image'
     requires_fs = True
+    preview_url_to_interpolate = '/imagehandler/<[explorationId]>/<[filepath]>'
```

Comment:
Probably want to declare this property (and give a docstring for it) in the base class, too.